template/docs/authentication.md

```diff
@@ -0,0 +1,3 @@
+# Authentication
```

Comment:
can we combine Authentication and Data Model into a single file? That would serve as a user-facing integration doc that we can easily include in `docs` to publish to support site.

Reply:
Sure. We can do that. What do you think about placing that under perhaps an `integration.md` file?

Reply:
or maybe `jupiterone.md`?